### DISCUSSION TITLE
Copilot doesn't work within already closed JSDoc comments

### DISCUSSION THREAD

**Discussion by asportnoy** [community/community#42497] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Typing `/**` in a JS file will automatically set up the closing `*/`. When typing within that already closed comment, Copilot will not give any suggestions for the content. Removing the `*/` fixes the issue.

**Comment by iamcjwilliams** [comment]
@asportnoy Thank you for your feedback!
